pkg/aws/eni/eni_gc_test.go

```diff
@@ -5,17 +5,18 @@ package eni
 
 import (
 	"context"
+	"testing"
 	"time"
 
-	check "github.com/cilium/checkmate"
+	"github.com/stretchr/testify/require"
 
 	ec2mock "github.com/cilium/cilium/pkg/aws/ec2/mock"
 	"github.com/cilium/cilium/pkg/controller"
 	"github.com/cilium/cilium/pkg/inctimer"
 	"github.com/cilium/cilium/pkg/logging"
 )
 
-func waitForControllerRun(c *check.C, controller *controller.Manager, name string, expectedCount int64) {
+func waitForControllerRun(t *testing.T, controller *controller.Manager, name string, expectedCount int64) {
```

Comment:
Add `t.Helper` here while at it:

```suggestion
func waitForControllerRun(t *testing.T, controller *controller.Manager, name string, expectedCount int64) {
	t.Helper()

```